libs/components/layout/src/lib/modules/action-button/action-button-container.default.component.scss

```diff
@@ -30,10 +30,7 @@
   .sky-action-button-flex {
     display: block;
     padding: 0;
-    margin: calc(
-        var(--sky-compat-action-button-flex-margin, $sky-margin-double) * -1
-      )
-      0;
+    margin: -$sky-margin-double 0;
```

Comment:
SCSS recommends wrapping unary operators in parenthesis (e.g. `(-$sky-margin-double)`).
See: https://sass-lang.com/documentation/operators/numeric/#unary-operators

Reply:
Done